libraries/AP_Mount/AP_Mount_Tusuav.cpp

```diff
@@ -0,0 +1,1128 @@
+                        if (recording != _recording) {
+                            _recording = recording;
+                            GCS_SEND_TEXT(MAV_SEVERITY_INFO,  "%s recording %s", send_text_prefix, _recording ? "ON" : "OFF");
+                        }
+
+                        // do something when recieve tracking command 
+                        TrackingStatus tracking_status;
+                        if(msg.status.bits.is_tracking){
+                            switch(msg.quality){
+                                case MessageGimbalAngle::QualityEnum::Track_Quality_None:{
+                                    tracking_status = TrackingStatus::Lost;
+                                    break;
+                                }
+                                case MessageGimbalAngle::QualityEnum::Track_Quality_Weak:{
+                                    tracking_status = TrackingStatus::Searching;
+                                    break;
+                                }
+                                case MessageGimbalAngle::QualityEnum::Track_Quality_Normal:{
+                                    tracking_status = TrackingStatus::Tracking;
+                                    break;
+                                }
+                                default:{
+                                    tracking_status = TrackingStatus::Lost;break;
+                                }
+                            }
+                        }
+                        else{
+                            tracking_status = TrackingStatus::Stopped;
+                        }
+						// tracking status push to GCS
+                        if (tracking_status != _last_tracking_status) {
+                            _last_tracking_status = tracking_status;
+                            switch (tracking_status) {
+                            case TrackingStatus::Stopped:
+                                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s tracking OFF", send_text_prefix);
+                                break;
+                            case TrackingStatus::Searching:
+                                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s tracking searching", send_text_prefix);
+                                break;
+                            case TrackingStatus::Tracking:
+                                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s tracking ON", send_text_prefix);
+                                break;
+                            case TrackingStatus::Lost:
+                                GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s tracking Lost", send_text_prefix);
+                                break;
+                            }
+                        }
+                    }
+                    break;
+                }
+
+                case GimbalSubCmd::GimbalSubCmd_LaserDistance:{
+					// recieve laser rangefinder value in decimeters and save the value in meters
```

Comment:
little white space issue here (too much indenting)

Reply:
Thanks for pointing that out! We've adjusted the indentation here and reviewed similar formatting throughout.